server/modules/mobile/service/arenaMobileImport/jobs/recordsImportJob.js

```diff
@@ -0,0 +1,89 @@
+import * as ActivityLog from '@common/activityLog/activityLog'
+
+import * as Survey from '@core/survey/survey'
+import * as Record from '@core/record/record'
+import * as Node from '@core/record/node'
+import * as Validation from '@core/validation/validation'
+import * as PromiseUtils from '@core/promiseUtils'
+
+import Job from '@server/job/job'
+import BatchPersister from '@server/db/batchPersister'
+import * as RecordManager from '@server/modules/record/manager/recordManager'
+import * as ActivityLogManager from '@server/modules/activityLog/manager/activityLogManager'
+
+import * as ArenaSurveyFileZip from '@server/modules/mobile/service/arenaMobileImport/model/arenaSurveyFileZip'
+
+const NODES_INSERT_BATCH_SIZE = 10000
+
+export default class RecordsImportJob extends Job {
```

Comment:
this job is basically the same as server/modules/arenaImport/service/arenaImport/jobs/recordsImportJob.js